File: app/src/main/java/reschikov/geekbrains/androidAdvancedLevel/weatherApplication/data/opencage/Rate.java
package reschikov.geekbrains.androidadvancedlevel.weatherapplication.data.opencage;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Rate {

    @SerializedName("limit")
    @Expose
    private Integer limit;
    @SerializedName("remaining")
    @Expose
    private Integer remaining;
    @SerializedName("reset")
    @Expose
    private Integer reset;

    public Integer getLimit() {
        return limit;
    }

    public Integer getRemaining() {
        return remaining;
    }

    public Integer getReset() {
        return reset;
    }
}

File: app/src/main/java/reschikov/geekbrains/androidadvancedlevel/weatherapplication/data/network/model/data/opencage/Rate.kt
package reschikov.geekbrains.androidadvancedlevel.weatherapplication.data.network.model.data.opencage

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Rate(@SerializedName("limit") @Expose val limit: Int,
                @SerializedName("remaining") @Expose val remaining: Int,
                @SerializedName("reset") @Expose val reset: Int)